recipes/multimethod/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "multimethod" %}
+{% set version = "1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a5c04db3d34ceecd704166d4775eb04c224cb654f634679ac993eb61016b2c2d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
This should probably be `noarch: python`

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
Good call ! Thanks @synapticarbors